django/db/models/sql/where.py

```diff
@@ -150,6 +150,11 @@ def relabeled_clone(self, change_map):
         clone.relabel_aliases(change_map)
         return clone
 
+    def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize=False, for_save=False):
```

Comment:
my initial thought was maybe it would be fine to use `**kwargs` here, but maybe it's better not to so invalid parameters are detected?